doc/man3/SSL_CONF_cmd.pod

```diff
@@ -110,6 +110,11 @@ Attempts to use the file B<value> as the set of temporary DH parameters for
 the appropriate context. This option is only supported if certificate
 operations are permitted.
 
+=item B<-record_padding>
+
+Attempts to use B<value> as the block size for padding TLS 1.3 records
```

Comment:
Yup, I used that terminology because people are used to it in ciphers. I didn't want to use "record" or anything else that might cause confusion. Any other suggestions?